be/src/runtime/stream_load/stream_load_executor.cpp

```diff
@@ -111,6 +112,15 @@ Status StreamLoadExecutor::execute_plan_fragment(StreamLoadContext* ctx) {
                 ctx->write_data_cost_nanos = MonotonicNanos() - ctx->start_write_data_nanos;
                 ctx->promise.set_value(status);
 
+                std::vector<TabletSharedPtr> tablets;
+                for (auto& info : ctx->commit_infos) {
+                    TabletSharedPtr tablet = StorageEngine::instance()->tablet_manager()->get_tablet(info.tabletId);
```

Comment:
BrokerLoad and RoutineLoad and Spark Load also have the persist the commit txn.

Reply:
The commit persisting has been moved to `void TabletsChannel::add_chunk`, where the sender is closed and the delta_writer is committed. It could cover all load methods.